src/components/Toaster/index.tsx

```diff
@@ -122,6 +149,11 @@ const ToastContainer = (): JSX.Element => {
             toastClassName={localCss(styles.toast(theme))}
             autoClose={AUTOCLOSE_DELAY}
             icon={false}
+            newestOnTop={newestOnTop}
+            limit={limit}
+            className={className}
```

Comment:
Why ? 🤔

Reply:
@chambo-e why for which one? All of them? I added some options for customizing a little more this component plus we need to allow `className` for each components.

Reply:
className yeah, why would we need it ?